lib/publiccloud/provider.pm

```diff
@@ -263,18 +263,19 @@ sub terraform_apply {
     my $instance_type        = get_var('PUBLIC_CLOUD_INSTANCE_TYPE');
     my $image                = $self->get_image_id();
     my $ssh_private_key_file = '/root/.ssh/id_rsa';
+    my $name                 = get_var('PUBLIC_CLOUD_RESOURCE_NAME', 'openqa-vm');
 
     record_info('WARNING', 'Terraform apply has been run previously.') if ($self->terraform_applied);
 
     assert_script_run('cd ' . TERRAFORM_DIR);
     record_info('INFO', "Creating instance $instance_type from $image ...");
-    assert_script_run('terraform init', TERRAFORM_TIMEOUT);
+    assert_script_run('terraform init -no-color', TERRAFORM_TIMEOUT);
```

Comment:
you may want to use TF_CLI_ARGS instead 
https://www.terraform.io/docs/commands/environment-variables.html

Reply:
what do we gain? with TF_CLI_ARGS this would be

`TF_CLI_ARGS="-no-color" terraform init` which is even more text than just `terraform init -no-color`

Reply:
instead of this you can call once 
`script_run('export TF_CLI_ARGS="-no-color"');`
and remove it from every terraform call

Reply:
Isn't every script_run command creating a subshell?

Reply:
I think not.
I wasn't aware of TF_CLI_ARGS, but I would like to stick to pass the arguments.

Reply:
why ?

Reply:
Cause one script_run() call depends on a state of a other one. And I like to avoid that.